src/components/CompareResults/Retrigger/RetriggerConfigModal.tsx

```diff
@@ -29,6 +29,7 @@ function RetriggerCountSelect({
         defaultValue={0}
         label={label}
         sx={{ height: 32 }}
+        data-testid={`${prefix}-retrigger-select`}
```

Comment:
you don't probably need this, instead I believe you can use the label: https://testing-library.com/docs/queries/bylabeltext